src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/sentryApplicationDashboard.tsx

```diff
@@ -16,11 +18,32 @@ import BarChart from 'app/components/charts/barChart';
 import DateTime from 'app/components/dateTime';
 import EmptyMessage from 'app/views/settings/components/emptyMessage';
 import Link from 'app/components/links/link';
+import Tag from 'app/views/settings/components/tag';
 
 import space from 'app/styles/space';
 import {SentryApp, SentryAppWebhookRequest} from 'app/types';
 import {t} from 'app/locale';
 
+const ResponseCode = ({code}: {code: number}) => {
+  return (
+    <div>
```

Comment:
Do you need the extra div?

Reply:
Yeah without the div the Tag width is messed up because of the grid layout